spring-integration-gcp/src/main/java/org/springframework/integration/gcp/inbound/PubSubInboundChannelAdapter.java

```diff
@@ -112,4 +116,12 @@ public void setAckMode(AckMode ackMode) {
 		Assert.notNull(ackMode, "The acknowledgement mode can't be null.");
 		this.ackMode = ackMode;
 	}
+
+	public void setMessageConverter(MessageConverter messageConverter) {
+		getMessagingTemplate().setMessageConverter(messageConverter);
+	}
+
+	public MessageConverter getMessageConverter() {
+		return getMessagingTemplate().getMessageConverter();
```

Comment:
I think we don't need extra delegation to the `MessagingTemplate` since we don't use its goodies afterward.
Right we use its `send(Message)` lately, but definitely not conversion already.
I mean that it would be much clear to have a local `MessageConverter` property.

Just IMO and I can live if you decide to stay with this style